elastalert/ruletypes.py

```diff
@@ -1308,3 +1308,9 @@ def percentage_violation(self, match_percentage):
         if 'min_percentage' in self.rules and match_percentage < self.rules['min_percentage']:
             return True
         return False
+
+    def format_string(self, format_config, target_value):
```

Comment:
What do you think about moving this format_string() into a util class so that it doesn't get repeated here?

Reply:
Working on it

Reply:
Moved format_string() to util.py and added tests for the function accordingly